group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -8686,6 +8695,88 @@ public void testListGroups() {
         assertEquals(expectAllGroupMap, actualAllGroupMap);
     }
 
+    @Test
```

Comment:
It would be great if we could add a test similar to `testListGroups` which verifies that the committed offset works as expected.

Reply:
add `testDescribeGroupsUncommittedGroup`